prql-java/README.md

```diff
@@ -1,30 +1,8 @@
 # prql-java
 
-`prql-java` offers Java bindings to the `prql-compiler` Rust library. It exposes
-a Java native method `public static native String toSql(String query)`.
+`prql-java` is Java library for PRQL that is divided into the following modules:
 
-It's still at an early stage, and currently requires compiling locally, and
-isn't published to Maven. Contributions are welcome.
+- [prql-api](/PRQL/prql/tree/main/prql-java/prql-api) is Java bindings for PRQL
```

Comment:
FYI getting some failures with these:

https://github.com/PRQL/prql/actions/runs/4258100365/jobs/7408963770

I think we probably want just:

```suggestion
- [prql-api](prql-java/prql-api) is Java bindings for PRQL
```